proarc-common/src/main/resources/cz/cas/lib/proarc/common/workflow/profile/workflow.xsd

```diff
@@ -37,6 +37,7 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
     <xs:complexType name="jobDefinition">
         <xs:sequence>
             <xs:element name="worker" type="tns:workerDefinition" minOccurs="0" />
+            <xs:element name="model" type="tns:modelDefinition" minOccurs="0" maxOccurs="unbounded" />
```

Comment:
Komentář jsem přidal. 

Žádný element znamená, že z toho záměru nejde vytvořit žádný dokument v Proarcu. Nechal bych to tak, protože chtějí RDFlow používat i na dokumenty, které nepůjdou do části Proarcu (pro evidenci).